origami-lib/test/test_textimageinput.py

```diff
@@ -0,0 +1,77 @@
+import requests 
+
+# server running locally at port 8888
+url = 'http://127.0.0.1:8888/event'
+
+# text data to be sent
+data = {
+    'input-text-0': 'hello',
+    'input-text-1': 'world',
+    'input-text-2': 'Is this a sample question?',
+    'input-text-4': 'This will not be read'
+}
+# list of images to be sent
+image_list = [
+    'example1.png',
+    'example2.png'
+]
+
+def test_sendText():
+    data_copy = data.copy()
+    data_copy['test_number'] = 0
+    r = requests.post(url=url, data=data_copy)
+    assert(r.status_code == 200)
```

Comment:
In addition to the status code, also if we receive the expected content

Reply:
The checking for expected content is currently done on the server and reflected in the status code

Reply:
Then move it to this file :smile:

Reply:
I thought the test is to check that the function works properly, that is to say that the server would receive the data correctly. To move it back to the test file, it would have to send the images and text back in the response?